source/MaterialXGenShader/ShaderGraph.cpp

```diff
@@ -547,16 +549,6 @@ ShaderGraphOutputSocket* ShaderGraph::addOutputSocket(const string& name, const
     return ShaderNode::addInput(name, type);
 }
 
-void ShaderGraph::renameInputSocket(const string& name, const string& newName)
-{
```

Comment:
We dont need to rename input and output anymore. They have a seperate field "variable" holding the unique name to use in code.